LibHR/Utils/suN_utils.c

```diff
@@ -470,10 +470,23 @@ void cooling_SPN(suNg* g_out, suNg* g_in, suNg* g_tilde, int cooling){
     _suNg_expand( S, *g_in);
         
     for (int nbcool=0;nbcool<cooling;nbcool++){
-        _suNgfull_times_suNgfull_dagger(B, S, U_tilde);
+        //_suNgfull_times_suNgfull_dagger(B, S, U_tilde);
```

Comment:
When removing code, it's better to remove it rather than leaving commented-out chunks of old code. Otherwise over time the code becomes an unreadable mess of archaeology of previous implementations.